readme.txt

```diff
@@ -111,30 +111,42 @@ If you get stuck, you can ask for help in the [Plugin Forum](https://wordpress.o
+
+**New Features**
+
+* Adds two new safety filters to the subscriptions detached debug tool: `wc_stripe_detached_subscriptions_maximum_time` and `wc_stripe_detached_subscriptions_maximum_count`
+* Show a notice when editing an active subscription that has no payment method attached
+* Show payment methods sync status on the UI
+* Add the number of pending webhooks to the Account status section
+* New WooCommerce Debug Tool to list subscriptions without a payment method attached
+* Remove BNPL payment methods (Klarna and Affirm) when other official plugins are active
+
+**Important Fixes and Updates**
+
+* Fix - Fixes wrong price formatting in express checkout
 * Fix - Moves the existing order lock functionality earlier in the order processing flow to prevent duplicate processing requests
-* Add - Adds two new safety filters to the subscriptions detached debug tool: `wc_stripe_detached_subscriptions_maximum_time` and `wc_stripe_detached_subscriptions_maximum_count`
-* Add - Show a notice when editing an active subscription that has no payment method attached
 * Fix - Fixes a possible fatal error when trying to generate the order signature for a `WC_Order_Refund` object
-* Add - New WooCommerce Debug Tool to list subscriptions without a payment method attached
 * Fix - Fixes a possible error notice when the `payment_request` Stripe setting key is not defined
-* Fix - Prevent irrelevant payment method update requests to Stripe during checkout
-* Add - A notice to take user back to WC onboarding flow after connecting the Stripe account
-* Tweak - Deprecate wc_connect_* filters
-* Fix - Prevent text field reset while editing Optimized Checkout title
-* Update - Improvements to custom checkout fields support for express checkout
-* Tweak - Use the Database Cache for the Stripe Account Data
-* Update - Update filter names to use the wc_stripe_* prefix
-* Add - Show payment methods sync status on the UI
 * Fix - No such customer error when creating a payment method with a new Stripe account
 * Fix - Validate create customer payload against required billing fields before sending to Stripe
-* Update - Enhanced logging system with support for all log levels and improved context handling
-* Fix - Fixes wrong price formatting in express checkout
 * Fix - Require email address only for Stripe customer validation when request is from the Add Payment Method page
-* Fix - Set default values for custom field options
 * Fix - Enforce rate limiter for failed add payment method attempts
-* Update - Add the number of pending webhooks to the Account status section
 * Fix - Prevent "Undefined array key charges_enabled" PHP warning when determining live‑mode status
+* Add - A notice to take user back to WC onboarding flow after connecting the Stripe account
+* Update - Update filter names to use the `wc_stripe_*` prefix
+* Update - Improvements to custom checkout fields support for express checkout
```

Comment:
I think this should be close to the top of the list, probably after the order lock changes.